src/Evolution/ComputeTags.hpp

```diff
@@ -40,5 +40,51 @@ struct AnalyticCompute
             inertial_coords, time, AnalyticFieldsTagList{})));
   }
 };
+
+// @{
```

Comment:
This should not be necessary. If Doxygen gets confused by the template specialization you can document the forward declaration and `/// \cond` the specialization.

In fact, since you now template the `function` on the `FieldTypes` you don't need the template specialization anymore. You could do this instead:

```cpp
template <size_t VolumeDim, typename AnalyticSolutionTag, typename TagsList>
 struct ErrorsCompute : db::add_tag_prefix<::Tags::Error,
                          ::Tags::Variables<TagsList>>,
       db::ComputeTag {
   using base = db::add_tag_prefix<::Tags::Error, ::Tags::Variables<TagsList>>;
   using return_type = tmpl::type_from<base>;

   using argument_tags = tmpl::append<
       // ...
       TagsList>;

   template <typename AnalyticSolution, typename... ErrorTags, typename... FieldTypes>
   static constexpr void function(
       const gsl::not_null<Variables<tmpl::list<ErrorTags...>>*> errors,
       const AnalyticSolution& analytic_solution_computer,
       const tnsr::I<DataVector, VolumeDim, Frame::Inertial>& inertial_coords,
       const double time, const FieldTypes&... fields) noexcept {
     // ...
     EXPAND_PACK_LEFT_TO_RIGHT(
         helper(make_not_null(&get<ErrorTags>(*errors)), fields));
     // ...
   }
 };
```